pyproject.toml

```diff
@@ -14,7 +14,7 @@ keywords = [
     "trio",
 ]
 classifiers = [
-    "Development Status :: 3 - Alpha",
+    "Development Status :: 4 - Beta",
```

Comment:
Does this reflect the maturity status of Trio? AnyIO has been marked "Production" since its v1.0 release.

Reply:
We don't have a 1.0 out yet so I think it's the best representation.